packages/web-components/cypress/support/index.js

```diff
@@ -35,7 +35,7 @@ Cypress.on('window:before:load', (win) => {
         throw new Error(msg)
     })
 
-    cy.stub(win.console, 'warn', (msg) => {
-        cy.now('task', 'warn', msg)
-    })
+    // cy.stub(win.console, 'warn', (msg) => {
```

Comment:
Modal throws a deprecation warning now so this was failing the tests. I don't believe this is actually testing anything, just the console.error one does.